packages/create-plugin/src/commands/generate/update-go-sdk-and-packages.ts

```diff
@@ -55,3 +56,17 @@ function updateGoMod(exportPath: string): Promise<void> {
     });
   });
 }
+
+function getLatestSdkVersion(exportPath: string): Promise<string> {
+  return new Promise(async (resolve, reject) => {
+    // run go list SDK_GO_MODULE@latest to get the latest version number
+    const command = `go list -m -json ${SDK_GO_MODULE}@latest`;
+    exec(command, { cwd: exportPath }, (error, stdout) => {
+      if (error) {
+        reject();
```

Comment:
I think it would make sense to add a debug log message here (to be able to see the actual error in case it's necessary).